src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -193,21 +196,35 @@ bool PerformAudioPassThruRequest::PrepareResponseParameters(
     mobile_apis::Result::eType& result_code, std::string& info) {
   LOG4CXX_AUTO_TRACE(logger_);
 
+  bool result = true;
   ResponseInfo ui_perform_info(result_ui_, HmiInterfaces::HMI_INTERFACE_UI);
   ResponseInfo tts_perform_info(result_tts_speak_,
                                 HmiInterfaces::HMI_INTERFACE_TTS);
-  const bool result =
-      PrepareResultForMobileResponse(ui_perform_info, tts_perform_info);
+
+  PrepareResultForMobileResponse(ui_perform_info, tts_perform_info);
```

Comment:
@tkireva PrepareResultForMobileResponse calculate result for sending with response to mobile side. Why did you remove assignment to result?